libbeat/docs/shared-command-line.asciidoc

```diff
@@ -14,7 +14,7 @@ Overwrite an individual setting in the config file. For example:
 +
 ["source","sh"]
 ----------------------------------------------------------------------
-sudo ./packetbeat -c packetbeat.yml -E name=mybeat
```

Comment:
@liubin Do you want to update your PR with the proposal above or should we take over? Thanks a lot for bringing this up.